recipes/mmeter/all/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.15)
+project(mmeter LANGUAGES CXX)
+
+add_library(mmeter ${MMETER_SRC_DIR}/src/MMeter.cpp)
+target_include_directories(mmeter PUBLIC ${MMETER_SRC_DIR}/include)
+set_target_properties(mmeter PROPERTIES
+PUBLIC_HEADER ${MMETER_SRC_DIR}/include/MMeter.h
+    WINDOWS_EXPORT_ALL_SYMBOLS ON
+    CXX_EXTENSIONS OFF
+    CXX_STANDARD 17
```

Comment:
This should not be hard-coded. If Conan 1 fails because macos uses gnu98 by default, using

```
if not self.settings.compiler.cppstd:
   tc.cache_variable["CMAKE_CXX_STANDARD"] = self._min_cppstd
```

is preferable :)

Reply:
Thanks!
Fixed.